lib/command.js

```diff
@@ -1559,6 +1590,27 @@ Expecting one of '${allowedValues.join("', '")}'`);
     this.error(message, { code: 'commander.missingMandatoryOptionValue' });
   }
 
+  /**
+   * `Option` conflicts with another option.
+   *
+   * @param {Option} option
+   * @param {Option} conflictingOption
+   * @api private
+   */
+  _conflictingOptionPresent(option, conflictingOption) {
+    const getErrorMessage = (option) => {
+      const optionKey = option.attributeName();
+      const source = this.getOptionValueSource(optionKey);
+      if (source === 'env') {
+        return `environment variable '${option.envVar}'`;
+      }
+      return `option '${option.flags}'`;
+    };
+
+    const message = `error: ${getErrorMessage(option)} cannot be used with ${getErrorMessage(conflictingOption)}`;
+    this.error(message, { code: 'commander.conflictingOptionPresent' });
```

Comment:
Suggest drop the `Present`

Reply:
Done in https://github.com/tj/commander.js/pull/1678/commits/f7739debcc830d3017c98d4cbfb797814d1e815a